### DISCUSSION TITLE
Is it possible to have a progression ?

### DISCUSSION THREAD

**Discussion by clifinger** [dlemstra/magick-wasm#124] [General]
Short question in the title ...

**Comment by dlemstra** [comment]
Of what? I don't understand what you are asking me?

**Reply by clifinger** [reply]
Oh, I am sorry, long day yesterday!

I mean like the `-monitor` option with the cli, to show a progress bar client side with large PSD file conversion.

Thank you

**Reply by dlemstra** [reply]
The `-monitor` is not yet available in the api. Once that has been added I will respond here.

**Reply by clifinger** [reply]
Thank you, Btw it work like a charm to convert our PSD

**Reply by dlemstra** [reply]
The latest release contains this feature.
